tests/lib/prepare-restore.sh

```diff
@@ -654,6 +670,19 @@ restore_suite_each() {
     # restore test directory saved during prepare
     tests.backup restore
 
+    # Save all the installed packages and remove the new packages installed 
+    if os.query is-classic; then
+        distro_print_installed_packages > installed-final.pkgs
+        diff -u installed-initial.pkgs installed-final.pkgs | grep -E "^\+" | tail -n+2 | cut -c 2- > installed-new.pkgs
+
+        # shellcheck disable=SC2002
+        packages="$(cat test | tr "\n" " ")"
```

Comment:
the test file does not exist?

```suggestion
        packages="$(tr "\n" " " < installed-new.pkgs)"
```

Reply:
nice catch, last migration I updated the names and forgot that one, thanks